reana_cluster/backends/kubernetes/templates/deployments/workflow-controller-template.yaml

```diff
@@ -45,6 +45,14 @@ spec:
         image: {{WORKFLOW_CONTROLLER_IMAGE}}
         command: ["flask", "consume-job-queue"]
         imagePullPolicy: {{IMAGE_PULL_POLICY}}
+        volumeMounts:
```

Comment:
duplicated of line 25-32, can you remove?

Reply:
it's not a duplication, we also need the volume mount for the second container